src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js

```diff
@@ -181,7 +183,7 @@ function paymentFromComponent() {
       .indexOf('cancelTransaction') > -1
   ) {
     Logger.getLogger('Adyen').error(
-      `Shopper cancelled transaction for order ${session.privacy.orderNo}`,
+      `Shopper cancelled transaction`,
```

Comment:
why was the orderNo removed? The error message doesn't have any useful information for debugging in this case :(

Reply:
Because we no longer use session variables. This message is only triggered when a shopper cancels the a paypal transaction so it's not very important to the merchant to know